### PR TITLE
Add govind1996

### PR DESCRIPTION
**Pull request by govind1996**
Please Review this.

### CODE REVIEW

**Comment by palash25** [comment]
@govind1996 did you run the tests locally?

**Comment by palash25** [comment]
Just a tip never push a commit from your master branch. *Always keep the master clean*

File: _contributors-app/govind1996.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+work_experience: |
+ - Research Intern @[Jadavpur University](www.jaduniv.edu.in):  Worked in the field of Fuzzy Logic. Project was to create a web based Jigsaw puzzle to calculate IQ of people.[May 2018 - present]
+
+
+ 
+
+project:
+- |
+    - [Gender Recognition using lower part of face
+](https://github.com/govind1996/Gender-Recognition): Implemented a research paper on gender recognition using lower part of face.
```
Comment (knrt10) [inline _contributors-app/govind1996.md]: move this `]` up

**Comment by govind1996** [comment]
@palash25 I will take care of it from next time.

**Comment by govind1996** [comment]
@knrt10 @palash25 review this.

**Pull request merged**